src/distribution/mod.rs

```diff
@@ -88,6 +88,21 @@ pub trait ContinuousCDF<K: Float, T: Float>: Min<K> + Max<K> {
     /// assert_eq!(0.5, n.cdf(0.5));
     /// ```
     fn cdf(&self, x: K) -> T;
+
+    /// Returns the survival function calculated
+    /// at `x` for a given distribution. May panic depending
+    /// on the implementor.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use statrs::distribution::{ContinuousCDF, Uniform};
+    ///
+    /// let n = Uniform::new(0.0, 1.0).unwrap();
+    /// assert_eq!(0.5, n.sf(0.5));
+    /// ```
+    fn sf(&self, x: K) -> T;
```

Comment:
I wonder if it might be useful to provide a degenerate default implementation of `1 - self.cdf(x)`

Reply:
Yeah I had wondered the same thing, but I ran into an issue where since the cdf is returning a generic `T` there was no implementation for `{float} - T` when I wrote up the `1 - self.cdf(x)`

I don't have much experience with generics, but I think it could be an easy addition for someone who did!